pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick a specific option to print to `stdout`:
+
+`gum choose "{{option_1}}" "{{option_2}}" "{{option_3}}"`
+
+- Open an interactive prompt for the user to input a string with a specific placeholder:
+
+`gum input --placeholder "{{value}}"`
+
+- Interactvely pick an option which will exit if "No" is chosen:
+
+`gum confirm "{{Continue?}}" --default=false --affirmative "{{Yes}}" --negative "{{No}}" {{&& echo "Yes selected" || echo "No selected"}}`
+
+- Show a spinner while a command is taking place with text alongside:
+
+`gum spin --spinner {{dot|line|minidot|jump|pulse|points|globe|moon|monkey|meter|hamburger}} --title "{{loading...}}" -- {{command}}`
+
+- Format text to include emojis:
+
+`echo "{{:smile: :heart: hello}}" | gum format -t {{emoji}}`
```

Comment:
Why not like this? Not all subcommands accept taking the text from stdin.
```suggestion
`gum format -t {{emoji}} "{{:smile: :heart: hello}}"`
```

Reply:
sure, I see no issue